### PR TITLE
create checkAuth controller

### PR DESCRIPTION
**Pull request by manar-abed**
relates #12

### CODE REVIEW

File: server/controllers/auth/checkAuth.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { Response } from 'express';
+
+const checkAuth = (req: any, res: Response) => {
+  const { userInfo } = req;
+  res.status(200).json({ data: userInfo });
```
Comment (Raghad-Mezied) [inline server/controllers/auth/checkAuth.ts:5]: Since the default status code is 200, then you don't need to send it

**Review by Raghad-Mezied — CHANGES_REQUESTED** [review]
You need to add the router which handled by this controller

**Review by abedshamia — APPROVED** [review]

**Review by Mahmoud-Ahmad2 — APPROVED** [review]

File: server/controllers/auth/checkAuth.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { Response } from 'express';
+
+const checkAuth = (req: any, res: Response) => {
```
Comment (Raghad-Mezied) [inline server/controllers/auth/checkAuth.ts:3]: Use the Request type you did previously.

File: server/routes/authRoutes.ts [hunk 3]
```diff
@@ -1,8 +1,10 @@
 import express, { Router } from 'express';
-import { login } from '../controllers';
+import { login, checkAuth } from '../controllers';
+// import { userAuth } from '../middlewares/auth';
 
 const router: Router = express.Router();
 
 router.post('/login', login);
+router.get('/api/checkAuth', checkAuth);
```
Comment (Raghad-Mezied) [inline server/routes/authRoutes.ts]: You need to pass the middleware here

**Review by Raghad-Mezied — CHANGES_REQUESTED** [review]

**Review by Raghad-Mezied — APPROVED** [review]

**Pull request merged**